recipes-cockpit/cockpit/files/version/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": 0,
+
+    "tools": {
+        "pinger": {
+            "label": "Aero-get-version",
```

Comment:
Please merge two commits to single commit and change folder name to plugins.
Everything else LGTM.